core/src/main/scala/org/apache/spark/Accumulators.scala

```diff
@@ -47,7 +48,7 @@ import org.apache.spark.util.Utils
  * @tparam T partial data that can be added in
  */
 class Accumulable[R, T] private[spark] (
-    @transient initialValue: R,
+    @(transient @param @field) initialValue: R,
```

Comment:
My original fix was to remove the unneeded annotations, and to transform the parameter where the annotation was making sense into `private val`. But it looks so much like a common pattern, that I didn't change it.